tests/01/input.json

```diff
@@ -1 +1 @@
-{"game":{"id":"cd71a0ba-ac21-4843-b31c-feb902b4a83f","ruleset":{"damagePerTurn":"15","foodSpawnChance":"20","minimumFood":"1","name":"royale","shrinkEveryNTurns":"20"},"timeout":500},"turn":13,"board":{"width":11,"height":11,"food":[{"x":3,"y":6}],"hazards":[],"snakes":[{"id":"gs_r8GrHbWqp3mRFxpH8WMSSjV4","name":"BlackHole","body":[{"x":6,"y":1},{"x":6,"y":2},{"x":6,"y":3},{"x":6,"y":4},{"x":6,"y":5},{"x":5,"y":5}],"health":97,"latency":364,"head":{"x":6,"y":1},"length":6,"shout":"","squad":""},{"id":"gs_jkjkRrfrWWtGMmGXcJpYVQQd","name":"Salazar Slitherin","body":[{"x":7,"y":0},{"x":7,"y":1},{"x":7,"y":2},{"x":7,"y":3}],"health":89,"latency":149,"head":{"x":7,"y":0},"length":4,"shout":"8 4 7 1","squad":""},{"id":"gs_vRx6kygXBgP3vSHkJRcycRxG","name":"Prüzze v2","body":[{"x":5,"y":0},{"x":4,"y":0},{"x":3,"y":0},{"x":3,"y":1},{"x":4,"y":1}],"health":99,"latency":22,"head":{"x":5,"y":0},"length":5,"shout":"One move, t=0","squad":""},{"id":"gs_MT4fmPtHShxKSrpyXRfXTf94","name":"marrrvin","body":[{"x":2,"y":5},{"x":2,"y":4},{"x":2,"y":3},{"x":3,"y":3},{"x":4,"y":3}],"health":92,"latency":187,"head":{"x":2,"y":5},"length":5,"shout":"","squad":""}]},"you":{"id":"gs_vRx6kygXBgP3vSHkJRcycRxG","name":"Prüzze v2","body":[{"x":5,"y":0},{"x":4,"y":0},{"x":3,"y":0},{"x":3,"y":1},{"x":4,"y":1}],"health":99,"latency":22,"head":{"x":5,"y":0},"length":5,"shout":"One move, t=0","squad":""}}
\ No newline at end of file
+{"game":{"id":"cd71a0ba-ac21-4843-b31c-feb902b4a83f","ruleset":{"damagePerTurn":"15","foodSpawnChance":"20","minimumFood":"1","name":"royale","shrinkEveryNTurns":"20", "version": "1"},"timeout":500},"turn":13,"board":{"width":11,"height":11,"food":[{"x":3,"y":6}],"hazards":[],"snakes":[{"id":"gs_r8GrHbWqp3mRFxpH8WMSSjV4","name":"BlackHole","body":[{"x":6,"y":1},{"x":6,"y":2},{"x":6,"y":3},{"x":6,"y":4},{"x":6,"y":5},{"x":5,"y":5}],"health":97,"latency":364,"head":{"x":6,"y":1},"length":6,"shout":"","squad":""},{"id":"gs_jkjkRrfrWWtGMmGXcJpYVQQd","name":"Salazar Slitherin","body":[{"x":7,"y":0},{"x":7,"y":1},{"x":7,"y":2},{"x":7,"y":3}],"health":89,"latency":149,"head":{"x":7,"y":0},"length":4,"shout":"8 4 7 1","squad":""},{"id":"gs_vRx6kygXBgP3vSHkJRcycRxG","name":"Prüzze v2","body":[{"x":5,"y":0},{"x":4,"y":0},{"x":3,"y":0},{"x":3,"y":1},{"x":4,"y":1}],"health":99,"latency":22,"head":{"x":5,"y":0},"length":5,"shout":"One move, t=0","squad":""},{"id":"gs_MT4fmPtHShxKSrpyXRfXTf94","name":"marrrvin","body":[{"x":2,"y":5},{"x":2,"y":4},{"x":2,"y":3},{"x":3,"y":3},{"x":4,"y":3}],"health":92,"latency":187,"head":{"x":2,"y":5},"length":5,"shout":"","squad":""}]},"you":{"id":"gs_vRx6kygXBgP3vSHkJRcycRxG","name":"Prüzze v2","body":[{"x":5,"y":0},{"x":4,"y":0},{"x":3,"y":0},{"x":3,"y":1},{"x":4,"y":1}],"health":99,"latency":22,"head":{"x":5,"y":0},"length":5,"shout":"One move, t=0","squad":""}}
```

Comment:
I added `"version": "1"` to the `ruleset` object in each of these cause my test snake requires it to run

Reply:
That makes sense. It seems the extension I was using didnt account for that. Thanks for fixing it!

Reply:
Ya I have to do this all the time when I use the Firefox extension to download games lol so I was familiar with the error!